docs/performance_tuning.md

```diff
@@ -0,0 +1,105 @@
+* KEY_CPU_BIND_THREAD
+* KEY_CPU_THROUGHPUT_STREAMS
+
+Read about available parameters on [OpenVINO supported plugins](https://docs.openvinotoolkit.org/latest/_docs_IE_DG_supported_plugins_CPU.html).
+
+Example docker command with setting `KEY_CPU_THROUGHPUT_STREAMS` to `KEY_CPU_THROUGHPUT_NUMA`:
+
+```
+docker run --rm -d --cpus-cpuset 0,1,2,3 -v <model_path>:/opt/model -p 9001:9001 ie-serving-py:latest /ie-serving-py/start_server.sh ie_serving model \
+--model_path /opt/model --model_name my_model --port 9001 --grpc_workers 8  --nireq 1 \
+--plugin_config "{\"CPU_THROUGHPUT_STREAMS\": \"KEY_CPU_THROUGHPUT_NUMA\",\"KEY_CPU_THREADS_NUM\": \"4\"}"
+```
+
+## Inference execution threading 
+
+Starting with the version *2019 R2*, OpenVINO Model Server is using [TBB](https://software.intel.com/en-us/tbb) threading instead
+ of [OMP](https://www.openmp.org/).
+ 
+It gives better performance especially in scenarios with shared CPU and enabled multiple AI models.
+The model server with TBB will be more flexible and easier to deploy in environments with dynamic load and resource allocation.
+
+If you need to use OpenMP for inference execution, adjust [Dockerfile_intelpython](../Dockerfile_intelpython) and add `-DTHREADING=OMP`
+in DLDT compilation command. Refer to [DLDT documentation](https://github.com/opencv/dldt/tree/2019/inference-engine).
+
+## Multi worker configuration
+
+Starting from version 2019 R3, it's possible to run OpenVINO Model Server with multiple server workers. It enables concurrent processing of
+multiple clients requests. It employs OpenVINO asynchronous inference execution mode. 
+Consider running model server with multiple server workers, if you're interested in throughput optimization without  
+creating multiple OVMS instances. You can expect the best results especially, when running the workloads on the accelerators like HDDL.
+
+Number of servers workers can be set on server startup with parameters `grpc_workers` and `rest_workers`.
+To really make use of multiple server workers, you should also set number of inference parallel requests
+for model with `nireq` parameter. This parameter defines how many inference operations can run in parallel.
+
+You should have at least as many server workers as inference requests set in `nireq` parameter in your models 
+(**grpc_wokers >= nireq** and **rest_workers >= nireq**).
+
+**Note**: When using CPU plugin, the vertical scalability (adding mode cores to the OVMS instnace) is a bit less efficient 
+then horizontal (adding more smaller OVMS instances). Add more instances if you need to maximize the throughput and you
+don't have restrictions of the number of instances and their RAM usage.
+
+### HDDL accelerators
+
+A single worker on OpenVINO Model Server could use only a single VPU (Vision Processing Unit) from HDDL accelerators at a time.
+Multiple workers can increase the throughput. With parallel handling of the requests, it is possible to utilize all available VPUs.
```

Comment:
I don't know how detailed we want our explanation here, but it's more like all VPUs are utilized either way, but with single worker they're utilized very poorly (in my experiments it was less than 15% utilization on each VPU).
Multiple worker setup also utilizes all VPUs, but it's far more effecient (more than 95% utilization per VPU). 
So, it might be a little confusing to say _"A single worker on OpenVINO Model Server could use only a single VPU"_. 
I would understand it like for card with 8x VPUs, with single worker OVMS I get 1x VPU fully utilized and the other 7x VPUs not used at all.